### ISSUE TITLE
Use official JSON schema

### ISSUE THREAD

**Issue by aviks** [bobby-brennan/fhir-swagger#14]
Currently the schema is generated from examples, which does not seem to be comprehensive. In one example I found, the `Patient` schema has a `birthDate` attribute, but since it's not present in the example used (`patient-example-a.json`), it does not appear in the swagger file. 

You already have #5 which is similar, but I was wondering if doing directly to the published schema (https://www.hl7.org/fhir/fhir.schema.json.zip) might be best?

**Comment by rbren** [comment]
Definitely! I don't there was JSON Schema published when I first created this repo. 

I would happily accept a PR that added these schemas.
